components/cv_form/projects.tsx

```diff
@@ -135,30 +136,9 @@ export default function Projects({ fProps }: Props) {
                     </span>
                   </div>
                   <div className="md:flex-grow">
-                    <Field
-                      as="select"
-                      name={`projects[${projectsIndex}].from_month`}
-                      className="rounded-md w-1/2 mr-2"
-                    >
-                      <option />
-                      {Options.monthsOptions.map((month) => (
-                        <option value={month.value} key={month.value}>
-                          {month.label}
-                        </option>
-                      ))}
-                    </Field>
-                    <Field
-                      as="select"
-                      name={`projects[${projectsIndex}].from_year`}
-                      className="rounded-md w-1/3"
-                    >
-                      <option />
-                      {Options.yearsOptions.map((year) => (
-                        <option value={year} key={year}>
-                          {year}
-                        </option>
-                      ))}
-                    </Field>
+                    <DatePicker
+                      name={`projects[${projectsIndex}].date_start`}
+                    />
```

Comment:
Datepicker where we select year and month
<img width="766" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/32201336/219380158-c2629fc9-f586-46cc-a2b1-1324fe7889d8.png">